### PR TITLE
add configuration for goerli

### PR DESCRIPTION
**Pull request by ralph-pichler**
add factory address for chain id 5 (`goerli`)

https://goerli.etherscan.io/address/0x334394e8c891e77b1449084aadd659920bb25247#code

### CODE REVIEW

File: pkg/settlement/swap/chequebook/factory.go [hunk 1]
```diff
@@ -156,3 +156,12 @@ func (c *factory) ERC20Address(ctx context.Context) (common.Address, error) {
 	}
 	return erc20Address, nil
 }
+
+// DiscoverFactoryAddress returns the canonical factory for this chainID
+func DiscoverFactoryAddress(chainID int64) (common.Address, bool) {
+	if chainID == 5 {
+		// goerli
```
Comment (Eknir) [inline pkg/settlement/swap/chequebook/factory.go:163]: Log here: connect to default factory for goerli network

File: pkg/node/node.go [hunk 2]
```diff
@@ -175,15 +175,20 @@ func NewBee(addr string, swarmAddress swarm.Address, keystore keystore.Service,
...
-			return nil, errors.New("no known factory address")
+			var found bool
+			factoryAddress, found = chequebook.DiscoverFactoryAddress(chainID.Int64())
+			if !found {
+				return nil, errors.New("no known factory address")
+			}
 		} else if !common.IsHexAddress(o.SwapFactoryAddress) {
 			return nil, errors.New("invalid factory address")
+		} else {
+			factoryAddress = common.HexToAddress(o.SwapFactoryAddress)
```
Comment (Eknir) [inline pkg/node/node.go:188]: Log here: connect to passed in factory <address>

**Review by Eknir — APPROVED** [review]
lgtm. I tried it out and it recognizes the faucet for Goerli. Would be good to add some log-lines to inform the user on what is happening.

File: pkg/node/node.go (hunk 2 above)
Comment (ralph-pichler) [reply pkg/node/node.go:189]: introduced a log

File: pkg/settlement/swap/chequebook/factory.go (hunk 1 above)
Comment (ralph-pichler) [reply pkg/settlement/swap/chequebook/factory.go:163]: introduced a log

**Review by janos — APPROVED** [review]

**Pull request merged**